ibexa/rector/5.0/rector.php

```diff
@@ -6,20 +6,19 @@
  */
 declare(strict_types=1);
 
+use Ibexa\Contracts\Rector\Sets\IbexaSetList;
 use Rector\Config\RectorConfig;
 
-return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->paths(
-        [
-            __DIR__ . '/src', // see if it matches your project structure
-            __DIR__ . '/tests',
-        ]
-    );
-
-    // define set lists
-    $rectorConfig->sets(
-        [
-            __DIR__ . '/vendor/ibexa/rector/src/contracts/Sets/ibexa-50.php', // set list for upgrading to Ibexa DXP 5.0
-        ]
-    );
-};
+return RectorConfig::configure()
+                   ->withPaths(
+                       [
+                           __DIR__ . '/src', // see if it matches your project structure
+                           __DIR__ . '/tests'
+                       ]
+                   )
+                   ->withSets(
+                       [
+                           IbexaSetList::IBEXA_50->value // rule set for upgrading to Ibexa DXP 5.0
+                       ]
+                   )
+;
```

Comment:
Something wrong happen with code formatting here

Reply:
> Something wrong happen with code formatting here

@adamwojs could you be more specific? This is how PHPStorm reformats that code.

Reply:
This look more natural for me:

```
return RectorConfig::configure()
   ->withPaths(
       [
           __DIR__ . '/src', // see if it matches your project structure
           __DIR__ . '/tests'
       ]
   )
   ->withSets(
       [
           IbexaSetList::IBEXA_50->value // rule set for upgrading to Ibexa DXP 5.0
       ]
   )
```

but the real questions is rather if this file is cover by php-cs-fixer :wink:

Reply:
> the real questions is rather if this file is cover by php-cs-fixer

There's no fixer in recipes. It will be covered after adding this file to any package with configured fixer :wink: 

Fixed as 39ff3ca